exposed-dao/src/main/kotlin/org/jetbrains/exposed/dao/Entity.kt

```diff
@@ -221,12 +221,16 @@ open class Entity<ID : Comparable<ID>>(val id: EntityID<ID>) {
         }
     }
 
-    operator fun <TColumn, TReal> ColumnWithTransform<TColumn, TReal>.getValue(o: Entity<ID>, desc: KProperty<*>): TReal =
-        toReal(column.getValue(o, desc))
+    operator fun <TColumn: Any?, TReal: Any?> ColumnWithTransform<TColumn, TReal>.getValue(o: Entity<ID>, desc: KProperty<*>): TReal? =
+        column.getValue(o, desc)?.let(toReal)
 
-    operator fun <TColumn, TReal> ColumnWithTransform<TColumn, TReal>.setValue(o: Entity<ID>, desc: KProperty<*>, value: TReal) {
+    operator fun <TColumn: Any?, TReal: Any?> ColumnWithTransform<TColumn, TReal>.setValue(o: Entity<ID>, desc: KProperty<*>, value: TReal) {
         column.setValue(o, desc, toColumn(value))
     }
+    @JvmName("setValueNullable")
+    operator fun <TColumn: Any?, TReal: Any> ColumnWithTransform<TColumn, TReal>.setValue(o: Entity<ID>, desc: KProperty<*>, value: TReal?) {
+        column.setValue(o, desc, value?.let(toColumn) as TColumn)
+    }
```

Comment:
The tests you added in `ColumnWithTransformTest` pass without these changes because [the default upper bound (if there was none specified) is Any?](https://kotlinlang.org/docs/generics.html#generic-constraints:~:text=The%20default%20upper%20bound%20(if%20there%20was%20none%20specified)%20is%20Any%3F), meaning that `ColumnWithTransform` already supports nullable columns.

Reply:
Interesting.  I will take another look.

Reply:
You're right; I must have been setting my transformations up incorrectly until now.  In that case, I'd like to offer this additional test coverage; I don't believe transformations had any coverage until now.

Reply:
Additional test coverage is always welcome. Thank you.